docs/accept-a-maximum-number-of-orders-per-day/README.md

```diff
@@ -3,14 +3,15 @@
 * [task.json](../../tasks/accept-a-maximum-number-of-orders-per-day.json) (for import/export)
 * [Task script](./script.liquid)
 
-This task works by monitoring the number of orders created per day, and clearing the inventory for all in-stock items when the daily order limit is reached. Optionally, this task can restore inventory to its original levels at midnight the next day, or on demand.
+This task works by setting your inventory to zero when the order limit is reached. (Specifically, this means setting inventory levels to 0 for all items that have a greater-than-zero inventory level.) There are no popups, or any specific messaging - your inventory will simply be dropped to zero, and if your shop is configured to stop selling out-of-stock products, your customers will be prevented from making additional purchases.
 
 ## Default options
 
 ```json
 {
   "maximum_daily_orders__number_required": "10",
   "only_clear_inventory_for_products_with_this_tag": "",
+  "prepend_this_html_to_product_description__multiline": "",
```

Comment:
Get slightly more specific about the name for this option. Suggestion: rather than "Prepend this HTML to product description", use "Prepend this HTML to product description when the order limit is reached". It's super verbose, but that clarifying detail is important!